.github/workflows/reusable_dockerfile_pipeline.yml

```diff
@@ -150,9 +157,7 @@ jobs:
           provenance: false
           platforms: linux/amd64
           # Only push if the head and base repos match, meaning it is not a fork
-          # yamllint disable
-          push: ${{ github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name }}
-          # yamllint enable
+          push: true
```

Comment:
this line needs to be reverted otherwise forks will break again. The new above if condition doesn't, and shouldn't handle this case.

Reply:
reverted in [LN 151](uses: docker/build-push-action@v5)